container/bin/python3.11

```diff
@@ -0,0 +1 @@
+python3
```

Comment:
This folder seems to have been added by accident

Reply:
yeah the container folder was added, when i tested